client-v2/src/selectors/frontsSelectors.ts

```diff
@@ -267,11 +267,13 @@ const visibleArticlesSelector = createSelector(
   }
 );
 
+const defaultVisibleFrontArticles = {};
+
 const visibleFrontArticlesSelector = createSelector(
   [collectionVisibilitiesSelector, collectionSetSelector],
   (collectionVisibilities, collectionSet) => {
     if (collectionSet === 'previously') {
-      return {};
+      return defaultVisibleFrontArticles;
```

Comment:
This will prevent rerenders, as an object literal will have a different identity each time we return it.